media/css/firefox/new/quantum/_newsletter.scss

```diff
@@ -0,0 +1,231 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../../pebbles/includes/animations';
+@import '../../../pebbles/includes/lib';
+
+#newsletter-subscribe {
+    background: #fff;
+    border-top: 1px solid #d7d7db;
+    color: #000;
+    display: none;
+    padding: 40px 0;
+
+    .newsletter-title {
+        background: url('/media/img/firefox/new/quantum/icon-newsletter.svg') center top no-repeat;
+        @include background-size(120px 82px);
+        padding: 100px 0 20px 0;
+        text-align: center;
+    }
+
+    .newsletter-form-content {
+        width: 300px;
```

Comment:
Should this be `280px`? Looks a bit off at 320px wide viewport.

Reply:
Nice catch, will update